plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c

```diff
@@ -11,15 +11,30 @@
 
 #include "fu-tpm-eventlog-device.h"
 
+struct FuPluginData {
+	GPtrArray		*pcr0s;
+};
+
 void
 fu_plugin_init (FuPlugin *plugin)
 {
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
+	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_RUN_BEFORE, "uefi");
```

Comment:
Is this something that we should concern turning into a circular dependency?
UEFI has this:
```
        fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "tpm_eventlog");
```

Reply:
> Is this something that we should concern turning into a circular dependency

Good question. The `METADATA_SOURCE` rule doesn't affect plugin startup order and so this should be fine.